### ISSUE TITLE
Keybind oddities

### ISSUE THREAD

**Issue by TherminatorX** [SlimeKnights/TinkersConstruct#1500]
Not really sure how to explain it, but keybinds behave funny when held down and the camera moves.

A good example would be to put on the Trav. Goggles and hold down Z. Everything works fine until you move your mouse. It will then switch between zoomed and un-zoomed at a rate that will almost give you vertigo. Almost.

This could be tied to the same issue that #1471 is having, but this is much easier to see.

(Note: If you have epilepsy, do not apply Night vision and hold down the key at night while moving)

**Comment by MrKunji** [comment]
The key is to ~toggle~ between zoom and normal vision, it's not for holding.

**Comment by TherminatorX** [comment]
@MrKunji Bugs found via unconventional means are still bugs.

**Comment by TherminatorX** [comment]
@bonii-xx I just had the idea that this might be a new Forge issue, not a TiC freak-out. Is this possible?

**Comment by Hydryad** [comment]
Using Regrowth modpack version 0.7.4, which has TConstruct-1.7.10-1.8.3b of the mod, I can reproduce the issue exactly as described. It is incredibly irritating, the slightest mouse movement while toggling the traveller's belt will result in it glitching out. If I press the toggle binding and hold it down, without moving the mouse, it will switch once and that is it. If you hold the button and move the mouse around, it freaks out and switches extremely rapidly. 

Basically, the issue is exactly as TherminatorX describes, just adding that I can reproduce it in this particular mod configuration.

**Comment by Hirsu** [comment]
I'd like to add a +1 to this issue, with the added problem my keybinds don't work consistently.
I actually need to press the keys multiple times to switch my hotbar, zoom, apply night vision etc.

Somehow, if I move my mouse while pressing the keys, I have a higher chance for it to succeed.
It's still very random though, and a major nuisance when I need to switch between my combat and tools hotbar quickly :)

Has anybody found the cause for this issue yet?
